charts/vela-core/crds/core.oam.dev_workflows.yaml

```diff
@@ -34,150 +34,16 @@ spec:
             type: string
           metadata:
             type: object
-          steps:
```

Comment:
why the steps field changed?

Reply:
This field is not changed, it's just the diff displayed by github.